aviary/subsystems/propulsion/test/test_propeller_performance.py

```diff
@@ -574,5 +575,95 @@ def test_tipspeed(self):
         assert_check_partials(partial_data, atol=5e-4, rtol=1e-4)
 
 
+class SquareRatioTest(unittest.TestCase):
+    """
+    Test the computation of square ratio with a maximum
+    """
+
+    def test_sqa_ratio_1(self):
+        prob = om.Problem()
+        prob.model.add_subsystem(
+            "group",
+            AreaSquareRatio(num_nodes=3, smooth_sqa=False),
+            promotes=["*"],
+        )
+        prob.setup(force_alloc_complex=True)
+        prob.set_val("DiamNac", val=2.8875, units='ft')
+        prob.set_val("DiamProp", val=10.0, units='ft')
+        prob.run_model()
+
+        sqa_ratio = prob.get_val("sqa_array", units='unitless')
+        assert_near_equal(sqa_ratio, [
+            0.08337656, 0.08337656, 0.08337656], tolerance=1e-5)
```

Comment:
it looks like the sqa array is not dependent on time, is there a reason it is an array instead of a scalar?

Reply:
It is true that sqa array is not dependent on time, but a later component needs a sqa_array. In current main, there is a `convert_sqa` component in `propeller_performance.py`.